velox/functions/prestosql/tests/ArrayJoinTest.cpp

```diff
@@ -268,4 +268,12 @@ TEST_F(ArrayJoinTest, jsonTest) {
       false,
       true);
 }
+
+TEST_F(ArrayJoinTest, unknownTest) {
+  testArrayJoinNoReplacement<UnknownValue>(
+      {std::nullopt, std::nullopt, std::nullopt}, ","_sv, ""_sv);
+  testArrayJoinReplacement<UnknownValue>(
```

Comment:
This test case doesn't seem right. I'd expect all NULL values to be replaced with 'apple'.